librecad/src/main/qc_applicationwindow.cpp

```diff
@@ -3664,3 +3670,8 @@ void QC_ApplicationWindow::updateWindowTitle(QWidget *w)
         w->setWindowTitle(w->windowTitle()+m_qDraftModeTitle);
     }
 }
+
+void QC_ApplicationWindow::goto_wiki()
+{
+    QDesktopServices::openUrl(QUrl("http://wiki.librecad.org/index.php/Main_Page"));
```

Comment:
Not yet, limited time and the heat here delays my progress on troubleshooting the content transfer tool.
Hope I can finish this in my holidays in August/September.